railties/lib/rails/generators/rails/app/templates/bin/setup

```diff
@@ -22,8 +22,9 @@ chdir APP_ROOT do
   system 'ruby bin/rake db:setup'
 
   puts "\n== Removing old logs and tempfiles =="
+  FileUtils.mkdir_p('tmp')
```

Comment:
I wouldn't do this here, maybe rake tmp:clear should work if there's no tmp directory?